python/triton/runtime/autotuner.py

```diff
@@ -170,6 +160,50 @@ def kernel_call():
                 print(f"Autotuning failed with {e}")
             return [float("inf"), float("inf"), float("inf")]
 
+    def check_disk_cache(self, tuning_key, configs, bench_fn):
+        # We can't serialize prehooks, so just give up and run the benchmarks.
+        if any(cfg.pre_hook for cfg in configs):
+            bench_fn()
+            return
```

Comment:
Should we also return when `tuning_key` is empty?

Reply:
Done!